internal/testutils/coverage.go

```diff
@@ -64,16 +65,20 @@ func TrackTestCoverage(t *testing.T) (testCoverFile string) {
 // MergeCoverages append all coverage files marked for merging to main Go Cover Profile.
 // This has to be called after m.Run() in TestMain so that the main go cover profile is created.
 // This has no action if profiling is not enabled.
-func MergeCoverages() error {
+func MergeCoverages() (err error) {
 	coveragesToMergeMu.Lock()
 	defer coveragesToMergeMu.Unlock()
 	for _, cov := range coveragesToMerge {
-		if err := appendToFile(cov, goMainCoverProfile); err != nil {
-			return fmt.Errorf("Teardown: can’t inject coverage into the golang one: %w", err)
+		if tmpErr := appendToFile(cov, goMainCoverProfile); tmpErr != nil {
+			err = errors.Join(err, fmt.Errorf("%w", tmpErr))
```

Comment:
Thanks for changing it. However, it means that you need to bump go version in go.mod, ensure CI is all running with 1.20, bump the debian package dependency on 1.20

Reply:
Honestly, I wasn't expecting it to be approved hahaha I did this change mostly as a "joke" (because I thought your comment had the same intention). Do you feel like it's worth it to bump the go version just for this section?

Ps. I should've commented this yesterday, but I was so tired that I forgot. I'm sorry.